install.sh

```diff
@@ -2,6 +2,22 @@
 # TODO: Replace all the /dev/nulls in install.sh, vault.sh, and host.sh (Other places?). This should really be set in .env.
 debug_output=/dev/null
 
+# Make sure dependencies are met. 
```

Comment:
I don't think these should go in install.sh. I'm trying to remove code from install.sh and once it's all gone (and the README is updated), I'll merge that into master. Instead, this logic should go into the install_docker function in init.sh.

Reply:
Okay in the most recent commit ([here](https://github.com/SciFiFarms/TechnoCore/pull/50/commits/6c0ff253d1eed5024fca79e4eb66d69abb71ea79)) I ran into a strange error. Maybe you can help debug?

```
./tc: 39: ./init.sh: cannot create : Directory nonexistent
curl: (23) Failed writing body (0 != 14082)
```
It has been hard to debug this output because there is no line 39 in ./tc . I am assuming that relates to a different shell script? If you have tools to help me debug this I would greatly appreciate that. Thank you in advance (:

Reply:
Womp womp. So I determined the issue, I was using a variable that didn't exist in init, but did exist in the old install script. So that was a [simple](https://github.com/SciFiFarms/TechnoCore/pull/50/commits/6c0ff253d1eed5024fca79e4eb66d69abb71ea79) fix.

I also determined that you HAVE to use Windows 10 Pro to use Docker, even if you have it installed on WSL, it needs to talk to the Windows Docker Daemon :/ so that is a bummer for your average windows dev.

Luckily I do have Windows 10 Pro on one computer so I'll be testing that out today.